plugins/org.jboss.tools.openshift.client/.classpath

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-4.0.3.Final.jar"/>
+	<classpathentry kind="lib" path="lib/openshift-restclient-java-4.0.3.Final.jar"/>
```

Comment:
I believe that we also need a change in the pom.xml where we bump the client version 4.0.3->4.0.4